conf/locale/en/LC_MESSAGES/django.po

```diff
@@ -6125,6 +6125,10 @@ msgid "Data Download"
 msgstr ""
 
 #: lms/djangoapps/instructor/views/instructor_dashboard.py
+msgid "Export Grades"
+msgstr ""
+
+#: lms/djangoapps/instructor/views/instructor_dashboard.py
```

Comment:
...honestly i have no idea how these got into the commit in the first place. `.gitignore` contains the pattern `*.po*`. i'll get rid of these.